src/haddock/modules/analysis/caprieval/capri.py

```diff
@@ -645,7 +649,9 @@ def rearrange_ss_capri_output(
         # This means there were only "dummy" values
         return
     else:
-        write_nested_dic_to_file(data, output_name)
+        data2 = convert_row_to_column_table(data)
+        table = create_human_readable_table(data2)
+        Path(output_name).write_text(table)
```

Comment:
this should output its custom suffix also, how about:

```suggestion
        table, suffix = create_human_readable_table(data2)
        Path(output_name).with_suffix(suffix).write_text(table)
```

Reply:
I disagree here because the tables python module I am implementing only concerns generating the formatted text. It's the responsibility of who/whatever asks for that text to save it with the extension found more appropriate. I have updated the extension in the `caprieval` tables to `txt`, though.